operator/validator/event_handler.go

```diff
@@ -39,29 +40,47 @@ func splitBytes(buf []byte, lim int) [][]byte {
 
 // Eth1EventHandler is a factory function for creating eth1 event handler
 func (c *controller) Eth1EventHandler(logger *zap.Logger, ongoingSync bool) eth1.SyncEventHandler {
-	return func(e eth1.Event) ([]zap.Field, error) {
+	return func(e eth1.Event) (logs []zap.Field, err error) {
```

Comment:
don't we want to not save when the error was just DB err for example? so we could run it again?

Reply:
as discussed in Discord, it's a tough call, both ways are bad and this way requires less code changes & testings